### PR TITLE
test(web): change test images to ghcr from dockerhub

### PR DESCRIPTION
**Pull request by nandor-magyar**
Dockerhub images must go because of accidental rate limit.

### CODE REVIEW

File: web/crux-ui/e2e/registry.spec.ts [hunk 1]
```diff
@@ -69,7 +69,7 @@ test("Unchecked registry shouldn't search images", async ({ page }) => {
   await expect(page.locator('label[for=imageName]')).toContainText('Image name and tag')
 
   await clearInput(page.locator('input[name=imageName]'))
-  await page.locator('input[name=imageName]').type('nginx:latest:test')
+  await page.locator('input[name=imageName]').type('ghcr.io/dyrector-io/mirror/nginx:mainline-stable:test')
```
Comment (polaroi8d) [inline web/crux-ui/e2e/registry.spec.ts:72]: You can use the new consts here ( eg.: `NGINX_TEST_IMAGE`), no?

File: web/crux-ui/e2e/registry.spec.ts [hunk 2]
```diff
@@ -88,7 +88,7 @@ test("Unchecked registry shouldn't search images", async ({ page }) => {
   await expect(page.locator('button:text-is("Add")')).toBeVisible()
 
   await clearInput(page.locator('input[name=imageName]'))
-  await page.locator('input[name=imageName]').type('nginx:latest')
+  await page.locator('input[name=imageName]').type('ghcr.io/dyrector-io/mirror/nginx:mainline-stable')
```
Comment (polaroi8d) [inline web/crux-ui/e2e/registry.spec.ts:91]: Same here.

File: web/crux-ui/e2e/utils/projects.ts [hunk 3]
```diff
@@ -84,7 +81,7 @@ export const addImageToVersion = async (page: Page, projectId: string, versionId
   await page.locator('button:has-text("Add image")').click()
   await expect(page.locator('h4:has-text("Add image")')).toHaveCount(1)
 
-  const registry = await page.waitForSelector(`button:has-text("Docker Hub Library")`)
+  const registry = await page.waitForSelector(`button:has-text("ghcr for testing")`)
```
Comment (polaroi8d) [inline web/crux-ui/e2e/utils/projects.ts:84]: `ghcr for testing` can be a const?

**Review by polaroi8d — CHANGES_REQUESTED** [review]

File: golang/internal/helper/image/image_test.go [hunk 4]
```diff
@@ -77,15 +77,15 @@ func TestProtoRegistryUrlEmpty(t *testing.T) {
...
+	assert.Equal(t, "ghcr.io/dyrector-io/mirror/nginx:mainline-stable", name)
 
 	name, err = imageHelper.ExpandImageName("nginx:tag")
 	assert.NoError(t, err)
 	assert.Equal(t, "docker.io/library/nginx:tag", name)
 
 	name, err = imageHelper.ExpandImageName("my-reg.com/library/nginx")
 	assert.NoError(t, err)
-	assert.Equal(t, "my-reg.com/library/nginx:latest", name)
+	assert.Equal(t, "my-reg.com/library/ghcr.io/dyrector-io/mirror/nginx:mainline-stable", name)
```
[the remark this answers is not in the document]
Comment (nandor-magyar) [reply golang/internal/helper/image/image_test.go]: It is not.

File: web/crux-ui/e2e/registry.spec.ts (hunk 1 above)
Comment (nandor-magyar) [reply web/crux-ui/e2e/registry.spec.ts]: done

File: web/crux-ui/e2e/registry.spec.ts [hunk 5]
```diff
@@ -69,7 +69,7 @@ test("Unchecked registry shouldn't search images", async ({ page }) => {
   await expect(page.locator('label[for=imageName]')).toContainText('Image name and tag')
 
   await clearInput(page.locator('input[name=imageName]'))
-  await page.locator('input[name=imageName]').type('nginx:latest:test')
+  await page.locator('input[name=imageName]').type('ghcr.io/dyrector-io/mirror/nginx:mainline-alpine:test')
```
Comment (m8vago) [inline web/crux-ui/e2e/registry.spec.ts:72]: `NGINX_TEST_IMAGE`

**Review by m8vago — CHANGES_REQUESTED** [review]

File: web/crux-ui/e2e/utils/projects.ts (hunk 3 above)
Comment (nandor-magyar) [reply web/crux-ui/e2e/utils/projects.ts:84]: Can be, done.

File: web/crux-ui/e2e/registry.spec.ts (hunk 5 above)
Comment (nandor-magyar) [reply web/crux-ui/e2e/registry.spec.ts:72]: Changed

**Review by polaroi8d — APPROVED** [review]

**Review by m8vago — APPROVED** [review]

**Pull request merged**